pyvelox/pyvelox.h

```diff
@@ -168,4 +348,52 @@ inline void addVeloxBindings(pybind11::module& m, bool asLocalModule = true) {
   rowType.def("names", &RowType::names, "Return the names of the columns");
 }
 
+inline void addVectorBindings(
+    pybind11::module& m,
+    bool asModuleLocalDefinitions = true) {
+  namespace py = pybind11;
+  using namespace facebook::velox;
+  std::shared_ptr<memory::MemoryPool> pool = memory::getDefaultMemoryPool();
+  py::class_<BaseVector, std::shared_ptr<BaseVector>>(
+      m, "BaseVector", py::module_local(asModuleLocalDefinitions))
+      .def_static(
+          "from_list",
+          [pool](const py::list& list) mutable {
+            return pyListToVector(list, pool.get());
+          })
+      .def(
+          "__str__",
+          [](std::shared_ptr<BaseVector>& v) { return v->toString(); })
+      .def("__len__", &BaseVector::size)
+      .def(
+          "__getitem__",
+          [](std::shared_ptr<BaseVector>& v, int64_t idx) {
+            return getItemFromVector(v, idx);
+          })
+      .def(
+          "__setitem__",
+          [](std::shared_ptr<BaseVector>& v, int64_t idx, py::handle& obj) {
+            setItemInVector(v, idx, obj);
```

Comment:
I dont think this works when assigning a None type.

Reply:
Yes, good catch. Added support